### ISSUE TITLE
Problem on iOS with quoting posts

### ISSUE THREAD

**Issue by C1n** [BoardTools/QuickReply#66]
When quoting a user the time parameter in the quote tag is changed when that user is using iOS + safari. Example, when a user quotes someone normal behavior would be:

`[quote=Striker post_id=699836 time=1481464715 user_id=755]`

Whereas quote behavior changes on iOS devices using safari.

`[quote=Striker post_id=699836 time=<a href="tel:1481464715">1481464715</a> user_id=755]`

This occurs on a test board that is upgraded to phpBB 3.2.0-RC2 (from 3.1.15) when the option below is enabled.
```
Insert full quotes into the quick reply form:
Replace the standard behaviour of the “Reply with quote” button.
```
So when you click the quote button it inserts the code as 

`[quote=Striker post_id=699836 time=<a href="tel:1481464715">1481464715</a> user_id=755]`

**Comment by C1n** [comment]
Looks like the unfortunate side-effect of a Safari feature.

http://stackoverflow.com/questions/226131/how-to-disable-phone-number-linking-in-mobile-safari
https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/PhoneLinks/PhoneLinks.html#//apple_ref/doc/uid/TP40007899-CH6-SW1

I image more people will encounter this issue, I don't know if you can do something with this, but I thought to post it anyway.

**Comment by lavigor** [comment]
Hello, @C1n!
Do you encounter the same behavior using the standard quoting feature? (When quick reply's quote is disabled.)

**Comment by C1n** [comment]
No, it is specifically occurring when the option "Insert full quotes into the quick reply form" is enabled. 

So when you click on the quote button of a post it inserts the quote in the quick reply form. Only then is the data changed.

**Comment by lavigor** [comment]
@C1n could you try changing the line [here](https://github.com/BoardTools/QuickReply/blob/develop-2.0/styles/all/template/event/viewtopic_body_postrow_post_after.html) from `<div id="decoded_p{postrow.POST_ID}" style="display: none;">{postrow.DECODED_MESSAGE}</div>` to `<div style="display: none;"><pre  id="decoded_p{postrow.POST_ID}">{postrow.DECODED_MESSAGE}</pre></div>` and see if the problem continues please?

**Comment by C1n** [comment]
Seems to have fixed the issue.

**Comment by lavigor** [comment]
Fixed in 2.0.0-beta1.
